### PR TITLE
feat: limit 10 organizations per user

### PR DESCRIPTION
**Pull request by pranalidhanavade**
### What 

- Changes in POST `/orgs` Create a new Organization API. Implemented logic to restricted user to create more than 10 organizations.
- Changes in POST `/users/org-invitations/{invitationId}` accept/reject organization invitation API. Implemented logic to restrict user for accepting organization invitation if user is already associated with 10 organziations

### CODE REVIEW

**Review by bhavanakarwade — APPROVED** [review]
LG

**Review by tipusinghaw — APPROVED** [review]
LG

File: apps/organization/repositories/organization.repository.ts [hunk 1]
```diff
@@ -632,6 +632,16 @@ export class OrganizationRepository {
     }
   }
 
+  async countUserOrganizations(userId: string): Promise<number> {
```
Comment (tipusinghaw) [inline apps/organization/repositories/organization.repository.ts:635]:
```suggestion
  async userOrganizationCount(userId: string): Promise<number> {
```

**Review by tipusinghaw — APPROVED** [review]

File: apps/organization/repositories/organization.repository.ts (hunk 1 above)
Comment (pranalidhanavade) [reply apps/organization/repositories/organization.repository.ts]: Resolved

**Pull request merged**